README.md

```diff
@@ -1792,6 +1793,35 @@ Creates an object composed of the object properties predicate returns truthy for
 
 ## String
 
+[_.padStart and _.padEnd](#_padstart-and-_padend)
+
+### _.padStart and _.padEnd
```

Comment:
this does the actual link in a .md file, you should remove line 1796

Reply:
sorted :)